pythonforandroid/toolchain.py

```diff
@@ -71,7 +71,6 @@ def check_python_dependencies():
 import glob
 import shutil
 import re
-import imp
```

Comment:
I'm curious about that one, is this not used e.g. here https://github.com/kivy/python-for-android/blob/e8857390afd5525eeffe0d4391d9f2e8307d2038/pythonforandroid/toolchain.py#L539 ?

Edit:
OK never mind it's also imported elsewhere so basically twice, thanks!
https://github.com/kivy/python-for-android/blob/e8857390afd5525eeffe0d4391d9f2e8307d2038/pythonforandroid/toolchain.py#L79